### ISSUE TITLE
Crash in setGPUMode with some non-Asus graphics card

### ISSUE THREAD

**Issue by 3c71** [DarthAffe/RGBSyncPlus#3]
Not sure which one, I have one Inno3D 1080 TI and one EVGA 1080 TI FTW3 Hybrid.
The call to setGPUMode(..., 1) crashes on the second card, which I believe it the Inno3D (listed in second position in other apps).

**Comment by DarthAffe** [comment]
Crashes at the Call...Mode-function seem to be all somehow related to this issue (https://rog.asus.com/forum/showthread.php?97754-Access-Violation-Wrong-EnumerateMB-Result).
There's no way i can fix this since the error occurs inside the SDK and Access-Violation can't be handled from the calling site.
We can just hope that the new version of the SDK has fixed that problem, but there's still no documentation or anything like that available.

**Comment by 3c71** [comment]
Isn't it possible to check if the GPU card is actually Asus-made and if not, skip the setGPUMode altogether?

**Comment by DarthAffe** [comment]
The SDK returns the GPU as a aura-compatible and SDK controllable device - otherwise this function wouldn't be called

**Comment by 3c71** [comment]
Yep sorry, I didn't really check for that either, I simply commented out the whole block as I don't have any Asus GPU ;)
